tweet-api/tests/test_analyze.py

```diff
@@ -0,0 +1,25 @@
+import json
+from collections import OrderedDict
+import unittest
+
+import requests
+
+class TestStringMethods(unittest.TestCase):
+
+    def test_analyze(self):
+        with open("./sample_requests/request.json") as f:
+            # python 3.7 dicts are ordered now, but some other libraries like pandas don't respect that
+            # so it's safest to stick with an OrderedDict since the order of the features for our input vector matter
```

Comment:
oh why does the order matter? (in this specific case there is only one key anyway so it wouldn't matter, but I'd love to understand why in the general case!)

Reply:
Ignore this test directory entirely, I made a task in my initial long comment that I plan to update this. 

Just to rapidly test yesterday, I copied it from another project where "the order of the features for our input vector matter".